### DISCUSSION TITLE
Feature Request: Different maxRevisions value in different sections

### DISCUSSION THREAD

**Discussion by acantes** [craftcms/cms#11477] [Ideas]
Whilst the general config setting 'maxRevisions' works for most websites, in some cases it may be advantageous to have more control over the number of revisions possble for different sections.

For example, a website might have a news blog section where 50 revisions of entries should be stored because the entries are subject to change frequently. That website though might also use Feed Me to pull in hundreds if not thousands of entries daily in say a products section. It would be nice to set that products section to say 3 revisions to keep the entry import process snappy and the database size manageable. That's one use case, I'd imagine there may be others.

**Comment by darylknight** [comment]
Agreed! I hadn't considered the impact of revisions on Feed Me. Subscribing.
Currently, I just disable revisions entirely on things that get auto-imported, but fine-grain control could be really nice too.

**Comment by ryssbowh** [comment]
Agreed. In my instance I have one section that needs infinite revisions which potentially on a large website will clog up the system with useless revisions for all the other sections.
